### DISCUSSION TITLE
2.8.2

### DISCUSSION THREAD

**Discussion by Karlatemp** [mamoe/mirai#1692] [Releases]
## mirai-core

- 修复使用绝对路径执行 `AbsoluteFolder.resolveXXX` 时会报错的错误 (#1687, #1689)

## mirai-console

- 修复出现日志名字为 `null` 的情况 (#1691)

<hr /><em>This discussion was created from the release <a href='https://github.com/mamoe/mirai/releases/tag/v2.8.2'>2.8.2</a>.</em>

**Comment by SekiBetu** [comment]
https://gitee.com/peratx/mirai-repo 仓库啥时候更新版本号，各种包的版本都没更新，mirai-api-http版本太旧甚至出现了兼容问题
